pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -290,6 +295,7 @@ def transfer_distrib_spawn_state_on_fit_end(self, results):
             self.mp_queue.put(best_model_path)
             self.mp_queue.put(last_path)
             self.mp_queue.put(results)
+            self.mp_queue.put(self._dict_to_float(self.lightning_module.trainer.logger_connector.callback_metrics))
```

Comment:
I believe this should be something the user can select by himself as a property or hook.